templates/includes/navigation.yaml

```diff
@@ -16,6 +16,14 @@ root:
 
 core:
   get-started:
+    installation-medias:
```

Comment:
Shouldn't these follow the order of what's mentioned on the `/core/get-started` page? Or are you planning to change that page to reflect this new order?